Corona-Warn-App/src/test/java/de/rki/coronawarnapp/covidcertificate/person/ui/overview/PersonOverviewViewModelTest.kt

```diff
@@ -259,6 +260,28 @@ class PersonOverviewViewModelTest : BaseTest() {
         }
     }
 
+    @Test
+    fun `openAdmissionScenarioScreen - success`() {
```

Comment:
These 2 tests are fine. However, I like it a bit better if we would call `triggerNow()` in these tests and verify if `dccWalletInfoCalculationManager.triggerNow()` was called and the correct scenarioId was passed. Then we wouldn't need "@VisibleForTesting" for `admissionScenarioId()` .

Reply:
this function is not triggering DccWalletInfo ,it calculates the scenarios, but I will verify that :)

Reply:
updated tests

Reply:
Thanks!